#include <iostream>
#include <stdexcept>
using std::cin; using std::cout; using std::endl; using std::runtime_error;

int main(void)
{
	int i, j;
	while (cin >> i >> j)
	{
		try
		{
			if (j == 0)
				throw runtime_error("divisor is 0");
			cout << i / j << endl;
			break;
		}
		catch (runtime_error err)
		{
			cout << err.what() << "\n";
			break;
		}
	}
	system("pause");
	return 0;
}

